### PR TITLE
Update copyrights to 2021 year

### PR DESCRIPTION
**Pull request by homksei**

### CODE REVIEW

**Comment by homksei** [comment]
/intelci: run
set infra_branch=dev/alekseik/copyrights_21

File: .ci/env/tbb.bat [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @echo off
 rem ============================================================================
-rem Copyright 2020 Intel Corporation
+rem Copyright 2021 Intel Corporation
```
Comment (averbukh) [inline .ci/env/tbb.bat:3]: It should be "2020-2021"

File: .ci/scripts/clang-format.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #! /bin/bash
 
 #===============================================================================
-# Copyright 2020 Intel Corporation
+# Copyright 2021 Intel Corporation
```
Comment (averbukh) [inline .ci/scripts/clang-format.sh:4]: It should be "2020-2021"

**Review by averbukh — CHANGES_REQUESTED** [review]
"2020" should become "2020-2021", "...-2020" should become "...-2021"

File: .ci/scripts/clang-format.sh (hunk 2 above)
Comment (homksei) [reply .ci/scripts/clang-format.sh:4]: For **cpp/daal/src/algorithms/dtrees/forest/regression/oneapi/df_regression_train_hist_oneapi_impl.i**  also should be "2020-2021" ?

Comment (averbukh) [reply .ci/scripts/clang-format.sh]: For every file, "2020" should become "2020-2021", "...-2020" should become "...-2021"

**Review by outoftardis — APPROVED** [review]
looks ok to me

**Comment by homksei** [comment]
/intelci: run
set infra_branch=dev/alekseik/copyrights_21

**Comment by homksei** [comment]
/intelci: run
set infra_branch=dev/alekseik/copyrights_21

**Review by averbukh — APPROVED** [review]

**Comment by homksei** [comment]
/intelci: run
set infra_branch=dev/alekseik/copyrights_21

**Review by napetrov — APPROVED** [review]

**Pull request merged**